bot.py

```diff
@@ -43,6 +43,18 @@
     """
     logger.info("pinged by {}".format(ctx.author))
     await ctx.send("pong")
+    
+@bot.command()
+async def wiki(ctx, wikipage: str):
```

Comment:
`wikipage` will only work if there's one word after .wiki because of how discord.py command parsing works. You can do this though:
```suggestion
async def wiki(ctx, *, wikipage: str):
```
which will mean it will contain *everything* the user says, verbatim